ts/sagas/wallet.ts

```diff
@@ -422,6 +450,8 @@ function* completionHandler(_: PaymentRequestCompletion) {
   // -> it should proceed with the required operations
   // and terminate with the "new payment" screen
 
+  // PIN REQUEST COULD BE HERE
```

Comment:
what does this comment mean?

Reply:
With this PR the pin is requested after the first summary. The pin could be requested to the user after the selection of the payment method instead. To do this change  the code could be transferred  in the completionHandler function